src/rules/prefer-block-comments-for-declarations.ts

```
@@ -22,6 +22,16 @@ export const preferBlockCommentsForDeclarations: TSESLint.RuleModule<
         return false;
       }
 
+      // Ignore ESLint directive comments
+      const commentText = comment.value.trim();
+      if (commentText.startsWith('eslint-disable') ||
```

ESLint

[error] Insert `⏎········`

(prettier/prettier)